code/src/components/Summary.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Summary = ({ result }) => (
+  <div className="summary">
+    <h2 className="summary-message">Hello {result[0]}!</h2>
+    <p className="summary-text">
+      Your first answer about <strong>The persistence of memory</strong> is
+      <span className="summary-answer">
+        <strong> {result[1]}</strong>
+      </span>
+    </p>
+    <p className="summary-text">
+      Your first answer about <strong>JW Rowling</strong> is
+      <span className="summary-answer">
+        <strong> {result[2]}</strong>
```

Comment:
Maybe you would like to add the actual answer along with the wrong/correct result :)